### ISSUE TITLE
Restart and Reopen Files Does Not Preserve Open with Options

### ISSUE THREAD

**Issue by bpotchik** [Vector35/binaryninja-api#3871]
After installing an update, files that were opened are reopened however if any were opened with options none of the open options are preserved.

**Comment by ccarpenter04** [comment]
I've recently experienced this in a VERY inconvenient way that resulted in almost none of the functions in the binary I was looking at being analyzed by default because the post compilation steps that were ran on it. It resulted in the functions either being too large or having too high of a complexity and as a result I had to manually go through each function and force the analysis which was almost as time consuming as the analysis phases themselves.

**Comment by bpotchik** [comment]
Yeah I ran into this recently as well. It would be really nice if it reopened with the settings you chose earlier.

As a side note, you can close the tab/file and reopen with options. If you have the `ui.files.restore.viewOptions` setting enabled it will remember the previous options set for that file.
